project-koku/masu
# matching on pod name matches for OCP Storage daily table

The OCP Storage daily summary table was doing a `LEFT JOIN` when determining what nodes a OpenShift volume is running on.  This was causing The node name of NULL to be allowed in the storage daily table.  I believe we should be a `JOIN` here since we only want to match when pod names are aligned between storage and usage.

Because of this the database had NULL values for node and was leading to the 500 seen here: https://github.com/project-koku/koku/issues/867

**Testing**
1. Ingest OCP data with matched pod name for storage and usage
2.  Verify that a group_by[node] && order_by[node] query is successful.
3.  Ingest data with misaligned pod names for storage and usage.
4.  Verify that a group_by[node] && order_by[node] query is successful.

**Test Results**
[koku_issue_867_ut.txt](https://github.com/project-koku/masu/files/3242694/koku_issue_867_ut.txt)
